### PR TITLE
Fix a bug in tex-input-validation.js

### PR DESCRIPTION
**Pull request by zhenyao**

### CODE REVIEW

**Comment by zhenyao** [comment]
@null77 Can you review?

File: sdk/tests/js/tests/tex-input-validation.js [hunk 1]
```diff
@@ -439,10 +439,14 @@ debug("");
 debug("Checking CopyTexSubImage2D: a set of inputs that are valid in GL but invalid in WebGL");
 
 testCases = [
+  // Section 3.8.5 of the GLES 3.0.3 spec, pg 140, requires that, if the
```
Comment (null77) [inline sdk/tests/js/tests/tex-input-validation.js]: Let's update this comment to the newest spec: 3.0.4 pg 141. (same section number)

**Comment by null77** [comment]
lgtm with comment fix!

File: sdk/tests/js/tests/tex-input-validation.js (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/js/tests/tex-input-validation.js]: Updated to 3.0.4, but it's actually pg 140 (starting there, but effective format is defined in 141)

**Pull request merged**